13/umbraco-engage/upgrading/upgrade-umbraco-engage.md

```diff
@@ -0,0 +1,49 @@
+# Upgrade Umbraco Engage
+
+This article shows how to upgrade Umbraco Engage to the latest version.
+
+When upgrading Umbraco Engage, be sure to consult the [version specific upgrade](version-specific-upgrade-notes.md) notes to learn about potential breaking changes and common pitfalls.
+
+{% hint style="warning" %}
+Before upgrading, it is recommended to backup your site and database.
+{% endhint %}
+
+## Get the latest version of Umbraco Engage
+
+To upgrade to the latest version of Umbraco Engage you can use:
+
+* NuGet
```

Comment:
You can add hyperlinks to the sections below.

Reply:
Yeah, I though about it, but since all sections are visible on the page at the same time, I don't see a reason to add the hyperlinks 🤞